plugins/lookup/filetree.py

```diff
@@ -54,41 +54,59 @@
 RETURN = r"""
   _raw:
     description: list of dictionaries with file information
+    type: list
+    elements: dict
     contains:
         src:
           description:
           - full path to file.
           - not returned when C(item.state) is set to C(directory).
+          type: path
         root:
           description: allows filtering by original location.
+          type: path
         path:
           description: contains the relative path to root.
+          type: path
         mode:
           description: The permissions the resulting file or directory.
+          type: str
         state:
           description: TODO
+          type: str
         owner:
           description: Name of the user that owns the file/directory.
+          type: raw
         group:
           description: Name of the group that owns the file/directory.
+          type: raw
         seuser:
           description: The user part of the SELinux file context.
+          type: raw
         serole:
           description: The role part of the SELinux file context.
+          type: raw
         setype:
           description: The type part of the SELinux file context.
+          type: raw
         selevel:
           description: The level part of the SELinux file context.
+          type: raw
         uid:
           description: Owner ID of the file/directory.
+          type: int
         gid:
           description: Group ID of the file/directory.
+          type: int
         size:
           description: Size of the target.
+          type: int
         mtime:
           description: Time of last modification.
+          type: int
         ctime:
           description: Time of last metadata update or creation (depends on OS).
+          type: int
```

Comment:
```suggestion
          description: Time of last metadata update or creation (depends on OS).
          type: int
```
100% int? Can't this be 'YYYY-MM-DD' or something?

Reply:
In the code `ret['ctime'] = st.st_ctime`, and where `st` is the result of https://docs.python.org/3/library/os.html#os.lstat, i.e. an stat_result object (https://docs.python.org/3/library/os.html#os.stat_result). The documentation is unfortunately not very clear.

I've checked typeshed, and interestingly it says float: https://github.com/python/typeshed/blob/master/stdlib/3/os/__init__.pyi#L242

I'll adjust the PR.